bigquery/snippets/snippet.go

```diff
@@ -425,9 +588,7 @@ func exportSampleTableAsCompressedCSV(client *bigquery.Client, gcsURI string) er
 	extractor := client.DatasetInProject(srcProject, srcDataset).Table(srcTable).ExtractorTo(gcsRef)
 	extractor.DisableHeader = true
 	// You can choose to run the job in a specific location for more complex data locality scenarios
-	// Ex: In this example, source dataset and GCS bucket are in the US
-	extractor.Location = "US"
-
+	// Ex: In this example, source dataset and GCS bucket are in the US	extractor.Location = "US"
```

Comment:
Did you mean to remove a line break here?

Reply:
Corrected.